backend/app/generic/models/presentable.py

```diff
@@ -7,21 +7,20 @@
 
 class Presentable(BaseModel):
     id: int | str
-    Resource_title_s: str
-    Description_s: str
-    Author_ss: Optional[list[str]] = None
-    Language_s: str
-    Keywords_ss: list[str]
-    License_s: str
-    Access_Rights_s: Optional[str] = "open_access"
-    Version_date__created_in__s: Optional[str] = None
-    Resource_Type_s: str
-    Content_Type_s: str
-    URL_s: str
-    EOSC_PROVIDER_s: str
-    Format_ss: list[str]
-    Level_of_expertise_s: Optional[str] = None
-    Target_group_s: Optional[str] = None
-    Qualification_s: Optional[str] = None
-    Duration_s: Optional[str] = None
-    facets: Optional[dict] = None
+    title: list[str]
+    description: list[str]
+    author_names: Optional[list[str]] = None
+    language: list[str]
+    keywords: list[str]
+    license: str
+    best_access_right: Optional[str] = "open_access"
+    publication_date: Optional[str] = None
+    resource_type: str
+    content_type: str
+    url: list[str]
+    eosc_provider: str
+    format: list[str]
```

Comment:
It is risky to assume that most of those fields exist. Many of them can be set to None